test/onnx/verify_onnx.cpp

```diff
@@ -1756,6 +1756,122 @@ TEST_CASE(resize_downsample_f_test)
     EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
 }
 
+TEST_CASE(resize_outsize_test)
+{
+    // resize using output_size input, rather than scales
+    migraphx::program p = migraphx::parse_onnx("resize_outsize_test.onnx");
+    p.compile(migraphx::make_target("ref"));
+
+    migraphx::shape sx{migraphx::shape::float_type, {1, 1, 2, 2}};
+    std::vector<float> dx(sx.elements());
+    std::iota(dx.begin(), dx.end(), 0.1f);
+
+    migraphx::shape sy{migraphx::shape::float_type, {1, 1, 4, 6}};
+    std::vector<float> dy(sx.elements(), 0);
+
+    migraphx::parameter_map pp;
+    pp["X"] = migraphx::argument(sx, dx.data());
+    pp["Y"] = migraphx::argument(sx, dy.data());
+
+    auto result = p.eval(pp).back();
+    std::vector<float> result_vector;
+    result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
+
+    // clang-format off
+    std::vector<float> gold = {0.1f, 0.1f, 1.1f, 1.1f, 1.1f, 1.1f, 
+                                2.1f, 2.1f, 3.1f, 3.1f, 3.1f, 3.1f, 
+                                2.1f, 2.1f, 3.1f, 3.1f, 3.1f, 3.1f, 
+                                2.1f, 2.1f, 3.1f, 3.1f, 3.1f, 3.1f};
+    // clang-format on
+
+    EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
+}
+
+TEST_CASE(resize_downsample_f_dyn_test)
+{
+    migraphx::onnx_options options;
+    options.default_dyn_dim_value = {1, 10};
+    options.use_dyn_output        = true;
```

Comment:
Why are you using this flag? AFAIK it is only used for `NonMaxSuppression`

Reply:
Then we can remove it.